app/controllers/v0/facilities/ccp_controller.rb

```diff
@@ -1,10 +1,26 @@
 # frozen_string_literal: true
 
 class V0::Facilities::CcpController < FacilitiesController
+  before_action :validate_params,
```

Comment:
I don't think you gain a lot by using these `only: SINGLE_CONTROLLER_ACTION` before actions. Its one more place to have to look in code to see what's going on. Leaving them in the controller method seems like it would be easier to understand

Reply:
this was a cargo cult from the VA controller.